### DISCUSSION TITLE
An unexpected error occurred while connecting Node(identifier=N0WeY6fqAl7K_-ND, uri=127.0.0.1:2333, status=NodeStatus.CONNECTING, players=0) to Lavalink: "127.0.0.1:2333/v4/websocket"

### DISCUSSION THREAD

**Discussion by Nartoleane** [EvieePy/Wavelink#305] [General]
the error is in title. i start lavalink, run the code, and this prints. idk what to do, ive looked everywhere.

**Comment by EvieePy** [comment]
The uri needs to be complete, E.g: `http://127.0.0.1:2333`

**Reply by Nartoleane** [reply]
same thing happens when i do that

**Comment by EvieePy** [comment]
Make sure the application.yml is in the same directory of your Lavalink. And make sure you are starting the Lavalink server via command line, E.g: `java -jar Lavalink.jar`

In the logs of Lavalink, if it says the port is `2333` (Or whatever you set it to) then it should be fine.

You can also try:
`http://localhost:2333`

**Reply by Nartoleane** [reply]
the application was in documents, thanks
